nucypher/blockchain/eth/signers.py

```diff
@@ -182,7 +185,7 @@ def sign_message(self, account: str, message: bytes, content_type: str = None, v
         else:
             raise NotImplementedError
 
-        return self.w3.manager.request_blocking("account_signData", [content_type, account, data])
+        return HexBytes(self.w3.manager.request_blocking("account_signData", [content_type, account, data]))
```

Comment:
Curious: What command / test / process did you use to test this - `nucypher ursula init`?

Reply:
i tested with `nucypher ursula run`, example traceback:
```
Traceback (most recent call last):
  File "/home/banteg/nu/venv/bin/nucypher", line 11, in <module>
    load_entry_point('nucypher', 'console_scripts', 'nucypher')()
  File "/home/banteg/nu/venv/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/banteg/nu/venv/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/banteg/nu/venv/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/banteg/nu/venv/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/banteg/nu/venv/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/banteg/nu/venv/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/banteg/nucypher/nucypher/cli/options.py", line 140, in wrapper
    return func(**kwargs)
  File "/home/banteg/nucypher/nucypher/cli/options.py", line 140, in wrapper
    return func(**kwargs)
  File "/home/banteg/nucypher/nucypher/cli/options.py", line 140, in wrapper
    return func(**kwargs)
  File "/home/banteg/nucypher/nucypher/cli/commands/ursula.py", line 354, in run
    json_ipc=general_config.json_ipc
  File "/home/banteg/nucypher/nucypher/cli/commands/ursula.py", line 255, in create_character
    start_learning_now=load_seednodes)
  File "/home/banteg/nucypher/nucypher/cli/actions.py", line 356, in make_cli_character
    **config_args)
  File "/home/banteg/nucypher/nucypher/config/node.py", line 234, in __call__
    return self.produce(**character_kwargs)
  File "/home/banteg/nucypher/nucypher/config/characters.py", line 109, in produce
    ursula = self.CHARACTER_CLASS(**merged_parameters)
  File "/home/banteg/nucypher/nucypher/characters/lawful.py", line 1003, in __init__
    self.log.debug(f"Created decentralized identity evidence: {self.decentralized_identity_evidence[:10].hex()}")
AttributeError: 'str' object has no attribute 'hex'
```